src/pages/ForgotPassword.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 
-import Navbar from '../components/presentationals/Navbar.jsx';
-import AuthenticationIllustration from '../components/presentationals/AuthenticationIllustration.jsx';
-import ForgotPasswordCard from '../components/presentationals/ForgotPasswordCard.jsx';
+import Navbar from '../components/presentationals/Navbar/Navbar';
+import AuthenticationIllustration from '../components/presentationals/AuthenticationIllustration/AuthenticationIllustration';
+import ForgotPasswordCard from '../components/presentationals/ForgotPassword/ForgotPasswordCard';
 import './Authentication.scss';
 
-const ForgotPassword = () => {
+const ForgotPassword = defaultAvatarUrl => {
```

Comment:
What is the difference between this component and `AuthenticationLayout`?

Reply:
The authentication layout houses sign up and sign in cards which is shown dynamically by a click of a tab button, but the forgot password card just holds the forgot password card